### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JarLowrey**
include some example SQL for people who aren't sure how to start, or if they are unsure if they are able to extract certain info using this database.

source: https://stackoverflow.com/questions/47423390/what-are-all-the-nutrients-in-the-usda-composition-api/47545386#47545386

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,5 +26,61 @@ I recommend using the open source [sqlitebrowser](http://sqlitebrowser.org/) to
...
+    INNER JOIN food ON nutrition.food_id=food.id 
+    INNER JOIN nutrient ON nutrition.nutrient_id=nutrient.id;
+```
+
+### Post processing
+
+You can export this data to JSON and do further processing, like indexing. Indexing speeds up access for client Applications and reduces file size (by removing redundant data). Indexing the final SQL example reduces JSON representation from ~10MB to 7.0MB.
+
+```js
+    import Amt from './exportedUsdaJsonDataFromSqliteOnlineFinalExample';
```
Comment (alyssaq) [inline README.md]: Whats contained in `exportedUsdaJsonDataFromSqliteOnlineFinalExample`? Each table in JSON?

File: README.md [hunk 2]
```diff
@@ -26,5 +26,61 @@ I recommend using the open source [sqlitebrowser](http://sqlitebrowser.org/) to
 ![sqllitebrowser_screenshot](
 https://github.com/sqlitebrowser/sqlitebrowser/raw/master/images/sqlitebrowser.png)
 
+You can also upload the created `*.sql3` file to https://sqliteonline.com/, saving yourself from needing to download any applications.
+
+## Usage Examples
+
+### SQL
+
+- `SELECT * FROM  nutrient;` - print out all nutrient data
```
Comment (alyssaq) [inline README.md]: I like these examples. Could you format them consistently? E.g. single space between words and a space after comma.
Thanks!

**Review by alyssaq — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JarLowrey) [reply README.md]: Nutrient amounts, it is the result of the final given query

**Comment by JarLowrey** [comment]
Updated from review. Do you happen to know why [nutrients are missing](https://github.com/alyssaq/usda-sqlite/issues/4) from the DB?

**Review by alyssaq — APPROVED** [review]

**Pull request merged**